config/routes.rb

```diff
@@ -9,6 +9,7 @@
     scope module: "jobs" do
       resources :scheduled, only: :index, as: :scheduled_jobs
       resources :failed, only: [:index, :show], as: :failed_jobs
+      resources :retries, only: [:update], as: :retry_jobs
```

Comment:
I feel like `create` is the more appropriate action here, as we do create a new retry. Also how do you feel about keeping it as `retries` instead of `retry_jobs`? 

```suggestion
      resources :retries, only: [:create]
```

Reply:
I agree. I started out this way but opted to go with update to make it easier to pass the param via the url. That said with button_to it's trivial to just pass it as a param.

Reply:
Oh right, I forgot about the param. You are right this would be more RESTful

Maybe we could do something like

```ruby
resources :jobs, only: :index do 
  resource :retry, only: :create, module: :jobs
end 

scope module: "jobs" do
  resources :scheduled, only: :index, as: :scheduled_jobs
  resources :failed, only: [:index, :show], as: :failed_jobs
end
```

This would change the jobs index route from `/jobs/all` to `/jobs` which is propbably better

Reply:
I like this! Just force-pushed an update and it seems to be working well this way